### PR TITLE
Utils docs

### PR DESCRIPTION
**Pull request by FalseR20**

### CODE REVIEW

File: src/sc_kpm/common/sc_keynodes.pyi [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+from sc_client.constants.sc_types import ScType
+from sc_client.models import ScAddr
+
+class ScKeynodes(dict):
+    _instance: ScKeynodes
+
+    def __new__(cls) -> ScKeynodes: ...
+    def __getitem__(self, identifier: str) -> ScAddr: ...
+    def resolve(self, identifier: str, sc_type: ScType = None) -> ScAddr: ...
```
Comment (AlexandrZagorskiy) [inline src/sc_kpm/common/sc_keynodes.pyi:15]: Why you propose to use .pyi file? I think it is not needed for our purposes

File: src/sc_kpm/common/sc_module.py [hunk 2]
```diff
@@ -35,13 +36,16 @@ class ScModule(ScModuleAbstract):
...
 
     def add_agent(
-        self, agent: Type[ScAgentAbstract], element: Union[str, ScAddr], event_type: ScEventType
+        self,
+        agent: Type[ScAgentAbstract],
+        element: Union[str, ScAddr] = QuestionStatus.QUESTION_INITIATED.value,
+        event_type: ScEventType = ScEventType.ADD_OUTGOING_EDGE,
     ) -> ScModuleAbstract:
         self._reg_params.append(RegisterParams(agent, element, event_type))
         return self
```
Comment (AlexandrZagorskiy) [inline src/sc_kpm/common/sc_module.py:45]: Add one more level of tabs, parameters tab level should be not equal to the function tab level

File: src/sc_kpm/utils/action_utils.py [hunk 3]
```diff
@@ -53,14 +52,14 @@ def execute_agent(
...
+    reaction: QuestionStatus = QuestionStatus.QUESTION_FINISHED_SUCCESSFULLY,
     wait_time: int = COMMON_WAIT_TIME,
-) -> bool:
+) -> Tuple[ScAddr, bool]:
     keynodes = ScKeynodes()
     question = call_agent(arguments, concepts, initiation)
     wait_agent(wait_time, question, keynodes[QuestionStatus.QUESTION_FINISHED.value])
     result = check_edge(sc_types.EDGE_ACCESS_VAR_POS_PERM, keynodes[reaction.value], question)
-    return result
+    return question, result
```
Comment (AlexandrZagorskiy) [inline src/sc_kpm/utils/action_utils.py:62]: rename _question_ to _question_node_, better to represent the element type in the name of the variable

File: src/sc_kpm/utils/action_utils.py [hunk 4]
```diff
@@ -120,9 +117,10 @@ def finish_action(action_node: ScAddr, status: QuestionStatus = QuestionStatus.Q
...
-        status_node = QuestionStatus.QUESTION_FINISHED_SUCCESSFULLY
-    else:
-        status_node = QuestionStatus.QUESTION_FINISHED_UNSUCCESSFULLY
-    finish_action(action_node, status_node)
     finish_action(action_node, QuestionStatus.QUESTION_FINISHED)
+    finish_action(
+        action_node,
+        QuestionStatus.QUESTION_FINISHED_SUCCESSFULLY
+        if is_success
+        else QuestionStatus.QUESTION_FINISHED_UNSUCCESSFULLY,
```
Comment (AlexandrZagorskiy) [inline src/sc_kpm/utils/action_utils.py:125]: For me, this is hard-to-read function calling. Looks like function takes 4 args, not 2

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

**Comment by FalseR20** [comment]
New PR: #8

**Pull request closed without merging**